embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/LogService.kt

```diff
@@ -0,0 +1,28 @@
+package io.embrace.android.embracesdk.internal.logs
+
+import io.embrace.android.embracesdk.LogExceptionType
+import io.embrace.android.embracesdk.Severity
+
+/**
+ * Creates log records to be sent using the Open Telemetry Logs data model.
+ */
+internal interface LogService {
+
+    fun log(
+        message: String,
+        severity: Severity,
+        properties: Map<String, Any>?
+    )
+
+    fun logException(
```

Comment:
Can these functions include KDoc? It should be similar to what already exists on `LogMessageService`.

Reply:
Added.